java/java-jersey-jaxrs/src/main/java/io/swagger/sample/data/PetData.java

```diff
@@ -23,8 +23,9 @@
 import java.util.*;
 
 public class PetData {
-  static List<Pet> pets = new ArrayList<Pet>();
-  static List<Category> categories = new ArrayList<Category>();
+
+  static List<Pet> pets = Collections.synchronizedList(new ArrayList<Pet>());
+  static List<Category> categories = Collections.synchronizedList(new ArrayList<Category>());
```

Comment:
I guess you don't need to synchronize this, as it is not used anywhere outside the static initializer.